apps/examples/testcase/le_tc/kernel/tc_libc_math.c

```diff
@@ -4213,21 +4213,35 @@ static void tc_libc_math_ynf(void)
  */
 static void tc_libc_math_gamma(void)
 {
-	double value;
+	double input[13] = {2.0, 3.0, INFINITY, 256.5,
+						1.0e-20, -257.5, -256.5, -1.0,
+						-1.5, 8.1, 8.3, 8.8, 9.3};
+	double compare[13] = {1, 2, (double)INFINITY, 0x1p1023 * input[3],
```

Comment:
is it necessary to type cast INFINITY as double ?

Reply:
modify